### PR TITLE
NestedFolders: Enable nested folders by default

### LINKED ISSUE #10339: Feat request: Nested folder
It would be awesome to have the ability to put folders in folders.

### PR DESCRIPTION
**Pull request by zserge**
🎉, I suppose?

P.S. I expect to merge this around Wednesday (March, 20th)

Fixes https://github.com/grafana/grafana/issues/10339

### CODE REVIEW

**Review by tolzhabayev — APPROVED** [review]
🚀

File: pkg/services/featuremgmt/registry.go [hunk 1]
```diff
@@ -274,6 +274,7 @@ var (
 			Description: "Enable folder nesting",
 			Stage:       FeatureStagePublicPreview,
```
Comment (tolzhabayev) [inline pkg/services/featuremgmt/registry.go:275]: need to change this to GA

Comment (tolzhabayev) [inline pkg/services/featuremgmt/registry.go:275]:
```suggestion
			Stage:       FeatureStageGeneralAvailability,
```

**Comment by rgarcia89** [comment]
What kind of permission are required for users to be able to create subfolders? I would expect that `Editor` on the root folder `Team A` should be enough, since they are also able to create dashboards within that folder. However, they are not seeing the button to create new folders. While I as admin am seeing it.

**Comment by papagian** [comment]
> What kind of permission are required for users to be able to create subfolders? I would expect that `Editor` on the root folder `Team A` should be enough, since they are also able to create dashboards within that folder. However, they are not seeing the button to create new folders. While I as admin am seeing it.

based on [this](https://github.com/grafana/grafana/blob/42126398bea2930a283284e9083ff479cebc5bd4/public/app/features/browse-dashboards/permissions.ts#L13) I think that the user need to have permission to create subfolders inside the folder
@joshhunt can you confirm?

**Comment by rgarcia89** [comment]
@papagian understood. But where is this permission coming from? It doesn't seem to be part of the Editor role. I have opened an issue for that https://github.com/grafana/grafana/issues/88120

**Pull request merged**